project/Build.scala

```diff
@@ -46,10 +49,12 @@ object Build extends sbt.Build {
   val algebirdVersion = "0.9.0"
   val chillVersion = "0.6.0"
 
+  val distDirectory = "output"
+
   override def projects: Seq[Project] = (super.projects.toList ++ BuildExample.projects.toList
       ++ Pack.projects.toList).toSeq
 
-  val commonSettings = Defaults.defaultSettings ++ Seq(jacoco.settings:_*) ++ sonatypeSettings  ++ net.virtualvoid.sbt.graph.Plugin.graphSettings ++
+  val commonSettings = Seq(jacoco.settings:_*) ++ sonatypeSettings  ++ net.virtualvoid.sbt.graph.Plugin.graphSettings ++
```

Comment:
Yes we will get an error about jetty. See in the thread reference above the following comments

![screen shot 2015-08-02 at 11 30 32 pm](https://cloud.githubusercontent.com/assets/146453/9031957/9d9b1cc6-396e-11e5-8672-fcb952224985.png)